module/VuFind/src/VuFind/RecordDriver/ProQuestFSG.php

```diff
@@ -83,7 +84,7 @@ public function setRawData($data)
      */
     public function getShortTitle()
     {
-        return html_entity_decode(strip_tags($this->marcGetShortTitle()));
+        return PropertyString::fromHtml($this->marcGetShortTitle());
```

Comment:
I am not clear if I need `html_entity_decode` anymore...?  Not seeing any rendering issues without it but I may be missing something.

Reply:
I strongly suspect we might need to add html_entity_decode to `PropertyString::fromHtml`, but I don't think we need it here. I think the simple thing would be to add a PropertyString test case that passes in `<i>Dungeons &amp; Dragons</i>` and confirms that the text output is `Dungeons & Dragons`. If that's broken as-is, then we may have another small PR on our hands... :-)